locksmith/src/controllers/v2/eventsController.ts

```diff
@@ -73,8 +79,8 @@ export const saveEventDetails: RequestHandler = async (request, response) => {
     request.user!.walletAddress
   )
 
-  // This was a creation!
-  if (created) {
+  // Only send email if event is deployed
+  if (created && event.status === EventStatus.DEPLOYED) {
```

Comment:
ok let's make sure the email is still sent _after_ it was deployed when this is happening later

Reply:
sure thing! good catch!